Don't check the libmagic version number, since it is determined incorrectly,
and this will cause the build to fail on Snow Leopard and earlier when the
otherwise unnecessary file port is not also installed, and upstream does not
plan to fix this.
https://bro-tracker.atlassian.net/browse/BIT-1096
--- CMakeLists.txt.orig	2013-11-07 12:27:11.000000000 -0600
+++ CMakeLists.txt	2013-11-22 20:12:59.000000000 -0600
@@ -75,10 +75,6 @@
 endif ()
 
 set(libmagic_req 5.04)
-if ( LibMagic_VERSION VERSION_LESS ${libmagic_req} )
-    message(FATAL_ERROR "libmagic of at least version ${libmagic_req} required "
-                        "(found ${LibMagic_VERSION})")
-endif ()
 
 include_directories(BEFORE
                     ${PCAP_INCLUDE_DIR}
